src/main/java/minicraft/entity/mob/ObsidianKnight.java

```diff
@@ -0,0 +1,343 @@
+package minicraft.entity.mob;
+
+import minicraft.core.Game;
+import minicraft.core.Updater;
+import minicraft.core.io.Localization;
+import minicraft.core.io.Sound;
+import minicraft.entity.Direction;
+import minicraft.entity.Entity;
+import minicraft.entity.FireSpark;
+import minicraft.entity.furniture.KnightStatue;
+import minicraft.gfx.Color;
+import minicraft.gfx.Font;
+import minicraft.gfx.Screen;
+import minicraft.gfx.SpriteLinker;
+import minicraft.item.Items;
+import minicraft.network.Analytics;
+import minicraft.screen.AchievementsDisplay;
+
+import java.util.Random;
+
+public class ObsidianKnight extends EnemyMob {
+	private static final SpriteLinker.LinkedSprite[][][] armored = new SpriteLinker.LinkedSprite[][][] {
+		Mob.compileMobSpriteAnimations(0, 0, "obsidian_knight_armored"),
+		Mob.compileMobSpriteAnimations(0, 2, "obsidian_knight_armored"),
+		Mob.compileMobSpriteAnimations(0, 4, "obsidian_knight_armored"),
+		Mob.compileMobSpriteAnimations(0, 6, "obsidian_knight_armored")
+	};
+	private static final SpriteLinker.LinkedSprite[][][] broken = new SpriteLinker.LinkedSprite[][][] {
+		Mob.compileMobSpriteAnimations(0, 0, "obsidian_knight_broken"),
+		Mob.compileMobSpriteAnimations(0, 2, "obsidian_knight_broken"),
+		Mob.compileMobSpriteAnimations(0, 4, "obsidian_knight_broken"),
+		Mob.compileMobSpriteAnimations(0, 6, "obsidian_knight_broken")
+	};
+	public static boolean beaten = false;
+	public static boolean active = false;
+	public static boolean failed = false;
+	private static boolean phase1 = true;
+	private static int attackPhaseCooldown = 0;
+	private static int attackPhase = 0;
+	/*
```

Comment:
More readable in my opinion if this comment is put above attackPhase

Reply:
Yes, I’ll do that, other variables should have comments for them too.

Reply:
Also, take a look at enums